proto/build.rs

```diff
@@ -50,6 +53,7 @@ static CUSTOM_TYPE_ATTRIBUTES: &[(&str, &str)] = &[
 static CUSTOM_FIELD_ATTRIBUTES: &[(&str, &str)] = &[
     (".celestia.core.v1.da.DataAvailabilityHeader.row_roots", VEC_BASE64STRING),
     (".celestia.core.v1.da.DataAvailabilityHeader.column_roots", VEC_BASE64STRING),
+    #[cfg(not(feature = "tonic"))]
```

Comment:
do we need this conditionally? I think it can hurt us too

Reply:
This was needed because I wanted to avoid the pbjson/prost::Any swicheroo (and this didn't seem strictly needed for tonic). Since we'll be moving to tendermint's Any everywhere, I think it can stay unconditional.